app/components/icons/library.tsx

```diff
@@ -875,3 +875,48 @@ export const ArrowLeftIcon = (props: SVGProps<SVGSVGElement>) => (
     />
   </svg>
 );
+
+export const EyeIcon = (props: SVGProps<SVGSVGElement>) => (
+  <svg
+    width=" 8%"
+    height=" 8%"
+    viewBox="0 0 24 24"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    {...props}
+  >
+    <path
+      d="M2.42012 12.7132C2.28394 12.4975 2.21584 12.3897 2.17772 12.2234C2.14909 12.0985 2.14909 11.9015 2.17772 11.7766C2.21584 11.6103 2.28394 11.5025 2.42012 11.2868C3.54553 9.50484 6.8954 5 12.0004 5C17.1054 5 20.4553 9.50484 21.5807 11.2868C21.7169 11.5025 21.785 11.6103 21.8231 11.7766C21.8517 11.9015 21.8517 12.0985 21.8231 12.2234C21.785 12.3897 21.7169 12.4975 21.5807 12.7132C20.4553 14.4952 17.1054 19 12.0004 19C6.8954 19 3.54553 14.4952 2.42012 12.7132Z"
+      stroke="currentColor"
+      stroke-width="2"
```

Comment:
This is not valid JSX. Please check your terminal and you will see the errors. EIther fix them manually or use something like this: https://react-svgr.com/playground/

Reply:
fixed